### PR TITLE
Add the GitGoat project

### PR DESCRIPTION
**Pull request by nir-valtman**
Please add the GitGoat project to the list.

### CODE REVIEW

File: _data/projects/GitGoat.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: GitGoat
+desc: GitGoat was built to enable DevOps and Engineering teams to design and implement a sustainable misconfiguration prevention strategy. It can be used for proof-of-concept projects, such as OpenSSF, that tend to identify security issues in code repositories. The cool thing about GitGoat is that it generates a complete set of repositories in your selected organization, then commits code from the last year, generates pull requests, merges them, and even defines specific repositoriy configurations. It is a great tool to learn how GitHub works and a facsinating opportunity to get into more of the internals of how Git actually works.
```
Comment (ritwik12) [inline _data/projects/GitGoat.yaml]: @nir-valtman Can you please make this short? 1-2 lines would be appreciated.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/GitGoat.yaml (hunk 1 above)
Comment (nir-valtman) [reply _data/projects/GitGoat.yaml]: @ritwik12 done. Please let me know if anything else is needed.

**Review by nir-valtman — COMMENTED** [review]
Shortened the description, as requested.

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@nir-valtman Thanks!!

**Pull request merged**